### PR TITLE
Minimal cross platform UI testing: Wallet and lao tests

### PR DESCRIPTION
**Pull request by 4xiom5**
- Created basic Wallet and Lao tests
- Moved all models and some utils from the backend test package to the common package
- Updated readme
- Renamed 'MockClient' functions from 'create<XXXX>' to 'generate<XXXX>'

### CODE REVIEW

**Review by simone-kalbermatter — CHANGES_REQUESTED** [review]
Looks really good! Just two small comments.

File: tests/karate/docs/README.md [hunk 1]
```diff
@@ -94,6 +94,37 @@ Feature: Create a Roll Call
     And match organizer.receiveNoMoreResponses() == true
 ```
 
+## Frontend tests architecture
+### Files
+* `utils`
+  * `constants.feature`: Contains all the necessary constants. Usually called at the beginning of any feature.
+  * `android.feature` and `web.feature`: Contain platform specific scenarios that will be used by the actual tests. Both files should implement the same scenarios.
+  * `platform.feature`: A simple wrapper around `android.feature` and `web.feature` that allows you to call the right scenario depending on the current env you are testing for. (i.e. if you set `karate.env=web`, if will call scenarios from `web.feature`)
```
[the remark this answers is not in the document]
Comment (4xiom5) [reply tests/karate/docs/README.md]: Fixed

File: tests/karate/docs/README.md [hunk 2]
```diff
@@ -68,8 +68,8 @@ Feature: Create a Roll Call
     * call read(serverFeature)
     * call read(mockClientFeature)
     * def organizer = call createMockFrontend
-    * def lao = organizer.createValidLao()
-    * def validRollCall = organizer.createValidRollCall(lao)
+    * def lao = organizer.generateValidLao()
```
[the remark this answers is not in the document]
Comment (4xiom5) [reply tests/karate/docs/README.md:71]: Added

**Review by simone-kalbermatter — APPROVED** [review]
LGTM!

**Pull request merged**